app/enterprise/0.34-x/kong-manager/configuration/authentication.md

```diff
@@ -7,42 +7,55 @@ chapter: 3
 > Before beginning, ensure completion of the steps in 
 [Getting Started with Kong Manager](/enterprise/{{page.kong_version}}/kong-manager/configuration/getting-started)
 
-Kong has the ability to verify the identity of all users with Basic 
-Authentication or LDAP Authentication. 
+Kong can verify the identity of all users with Basic 
+Authentication or LDAP Authentication Advanced. 
+
+⚠️ **IMPORTANT**: Before enabling authentication, ensure that you
+have at least one Super Admin account. You may have set one up during
+the [Quick Start](/enterprise/{{page.kong_version}}/getting-started/quickstart),
+or you can set one up on the Organization page of Kong Manager.
+
+## How to set up a Super Admin
+Before enabling authentication, create a Super Admin. You can
+skip this section if you created a Super Admin during initial
+Kong setup.
+
+1. Go to the Organization page in Kong Manager.
+
+2. Click +Invite User and fill out the form. Give the user
+the super-admin role in the default workspace.
+
+3. Go back to the Organization page, Invited section, and click
+the email address of the user in order to view the user.
+
+4. Click Generate Registration Link. Copy the link. You'll use
```

Comment:
"Generate Registration" link

Instead of "you'll use it later... your account", "It will be used later... the account".